resources/assets/js/angular/app.js

```diff
@@ -86,11 +86,11 @@ angular.module('fc', [
       }) 
       .state('root.admin.privacy', {
          url: 'privacy',
-         templateUrl: 'angular/admin/privacy.tpl.html'
+         templateUrl: 'templates/admin/privacy.tpl.html'
       })
       .state('root.admin.contact', {
          url: 'contact',
-         templateUrl: 'angular/admin/contact.tpl.html'
+         templateUrl: 'templates/admin/contact.tpl.html'
       });
 })
 .run(['$rootScope', '$state', 'userService', function ($rootScope, $state, userService) {
```

Comment:
From a PHP perspective, templates and code in the same place freaks me out. From a module perspective I see where you're coming from. We can play around with the globstar and a modules directory to allow for this and automatic addition of all files.